File: services/GroupService/src/main/java/com/se1853_jv/dto/response/HighlightResponse.java
package com.se1853_jv.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class HighlightResponse {
    private String id;
    private String color;
    private Long coordinationX;
    private Long coordinationY;
    private Integer pageNumber;
    private UUID paperId;
    private UUID collectionId;
    private UUID userId;
}

File: services/paper-service/src/main/kotlin/com/se1853_jv/dto/response/HighlightResponse.kt
package com.se1853_jv.dto.response

import java.util.UUID

data class HighlightResponse(
    val id: String,
    val color: String,
    val coordinationX: Long,
    val coordinationY: Long,
    val pageNumber: Int,
    val paperId: UUID,
    val collectionId: UUID,
    val userId: UUID
)







